conf/build.manifest

```diff
@@ -156,6 +159,9 @@ apt_preferences:
 - Package: "*polkit*"
   Pin: "release n=bullseye-security"
   Pin-Priority: 1000
+- Package: "*golang*"
```

Comment:
Can we try to keep this list alphabetical please?

Reply:
Sure, created a ticket for it.